### PR TITLE
Update mirrors to https://deb.debian.org/debian/ in formulas where ap…

### PR DESCRIPTION
**Pull request by thehonkinggoose**
This is a larger pull request updating mirrors in several formulas to use https://deb.debian.org/debian/ as recommended by the audit system rather than scattered across several mirror sources.

### CODE REVIEW

File: Formula/dns2tcp.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Dns2tcp < Formula
   desc "TCP over DNS tunnel"
   homepage "https://packages.debian.org/sid/dns2tcp"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/d/dns2tcp/dns2tcp_0.5.2.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/d/dns2tcp/dns2tcp_0.5.2.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/d/dns2tcp/dns2tcp_0.5.2.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/d/dns2tcp/dns2tcp_0.5.2.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/dns2tcp.rb]: Remove `mirror`

File: Formula/fakeroot.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Fakeroot < Formula
   desc "Provide a fake root environment"
   homepage "https://tracker.debian.org/pkg/fakeroot"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/f/fakeroot/fakeroot_1.23.orig.tar.xz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.23.orig.tar.xz"
+  url "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.23.orig.tar.xz"
+  mirror "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.23.orig.tar.xz"
```
Comment (fxcoudert) [inline Formula/fakeroot.rb]: Remove `mirror`

File: Formula/freealut.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class Freealut < Formula
   desc "Implementation of OpenAL's ALUT standard"
   homepage "https://github.com/vancegroup/freealut"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/f/freealut/freealut_1.1.0.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/f/freealut/freealut_1.1.0.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/f/freealut/freealut_1.1.0.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/f/freealut/freealut_1.1.0.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/freealut.rb]: Remove `mirror`

File: Formula/giflib.rb [hunk 4]
```diff
@@ -17,8 +17,8 @@ class Giflib < Formula
   # https://sourceforge.net/p/giflib/bugs/102/
   # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=820526
   patch do
-    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/g/giflib/giflib_5.1.4-3.debian.tar.xz"
-    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/g/giflib/giflib_5.1.4-3.debian.tar.xz"
+    url "https://deb.debian.org/debian/pool/main/g/giflib/giflib_5.1.4-3.debian.tar.xz"
+    mirror "https://deb.debian.org/debian/pool/main/g/giflib/giflib_5.1.4-3.debian.tar.xz"
```
Comment (fxcoudert) [inline Formula/giflib.rb]: Remove `mirror`

File: Formula/grib-api.rb [hunk 5]
```diff
@@ -1,8 +1,8 @@
 class GribApi < Formula
   desc "Encode and decode grib messages (editions 1 and 2)"
   homepage "https://software.ecmwf.int/wiki/display/GRIB/Home"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/g/grib-api/grib-api_1.27.0.orig.tar.xz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/g/grib-api/grib-api_1.27.0.orig.tar.xz"
+  url "https://deb.debian.org/debian/pool/main/g/grib-api/grib-api_1.27.0.orig.tar.xz"
+  mirror "https://deb.debian.org/debian/pool/main/g/grib-api/grib-api_1.27.0.orig.tar.xz"
```
Comment (fxcoudert) [inline Formula/grib-api.rb]: Remove `mirror`

File: Formula/icon-naming-utils.rb [hunk 6]
```diff
@@ -4,8 +4,8 @@ class IconNamingUtils < Formula
   # Upstream seem to have enabled by default SSL/TLS across whole domain which
   # is problematic when the cert is for www rather than a wildcard or similar.
   # url "http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.gz"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/i/icon-naming-utils/icon-naming-utils_0.8.90.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/i/icon-naming-utils/icon-naming-utils_0.8.90.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/i/icon-naming-utils/icon-naming-utils_0.8.90.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/i/icon-naming-utils/icon-naming-utils_0.8.90.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/icon-naming-utils.rb]: Remove `mirror`

File: Formula/iprint.rb [hunk 7]
```diff
@@ -1,8 +1,8 @@
 class Iprint < Formula
   desc "Provides a print_one function"
   homepage "https://www.samba.org/ftp/unpacked/junkcode/i.c"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/i/iprint/iprint_1.3.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/i/iprint/iprint_1.3.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/i/iprint/iprint_1.3.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/i/iprint/iprint_1.3.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/iprint.rb]: Remove `mirror`

File: Formula/jhead.rb [hunk 8]
```diff
@@ -21,8 +21,8 @@ class Jhead < Formula
   patch :DATA
 
   patch do
-    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/j/jhead/jhead_3.00-4.debian.tar.xz"
-    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/j/jhead/jhead_3.00-4.debian.tar.xz"
+    url "https://deb.debian.org/debian/pool/main/j/jhead/jhead_3.00-4.debian.tar.xz"
+    mirror "https://deb.debian.org/debian/pool/main/j/jhead/jhead_3.00-4.debian.tar.xz"
```
Comment (fxcoudert) [inline Formula/jhead.rb]: Remove `mirror`

File: Formula/jxrlib.rb [hunk 9]
```diff
@@ -1,8 +1,8 @@
 class Jxrlib < Formula
   desc "Tools for JPEG-XR image encoding/decoding"
   homepage "https://archive.codeplex.com/?p=jxrlib"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/j/jxrlib/jxrlib_1.1.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/j/jxrlib/jxrlib_1.1.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/j/jxrlib/jxrlib_1.1.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/j/jxrlib/jxrlib_1.1.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/jxrlib.rb]: Remove `mirror`

File: Formula/lcrack.rb [hunk 10]
```diff
@@ -1,8 +1,8 @@
 class Lcrack < Formula
   desc "Generic password cracker"
   homepage "https://packages.debian.org/sid/lcrack"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/l/lcrack/lcrack_20040914.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/l/lcrack/lcrack_20040914.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/l/lcrack/lcrack_20040914.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/l/lcrack/lcrack_20040914.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/lcrack.rb]: Remove `mirror`

File: Formula/libffi.rb [hunk 11]
```diff
@@ -2,8 +2,8 @@ class Libffi < Formula
   desc "Portable Foreign Function Interface library"
   homepage "https://sourceware.org/libffi/"
   url "https://sourceware.org/pub/libffi/libffi-3.2.1.tar.gz"
-  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/libf/libffi/libffi_3.2.1.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/libf/libffi/libffi_3.2.1.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/libf/libffi/libffi_3.2.1.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/libf/libffi/libffi_3.2.1.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/libffi.rb]: Remove the second `mirror` line

File: Formula/liblockfile.rb [hunk 12]
```diff
@@ -1,8 +1,8 @@
 class Liblockfile < Formula
   desc "Library providing functions to lock standard mailboxes"
   homepage "https://tracker.debian.org/pkg/liblockfile"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/libl/liblockfile/liblockfile_1.14.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/libl/liblockfile/liblockfile_1.14.orig.tar.gz"
+  url "https://deb.debian.org/debian/pool/main/libl/liblockfile/liblockfile_1.14.orig.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/libl/liblockfile/liblockfile_1.14.orig.tar.gz"
```
Comment (fxcoudert) [inline Formula/liblockfile.rb]: Remove `mirror`

File: Formula/libquicktime.rb [hunk 13]
```diff
@@ -20,8 +20,8 @@ class Libquicktime < Formula
   # Also, fixes from upstream for CVE-2017-9122 through CVE-2017-9128, applied
   # by Debian since 30 Jun 2017.
   patch do
-    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/libq/libquicktime/libquicktime_1.2.4-12.debian.tar.xz"
-    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/libq/libquicktime/libquicktime_1.2.4-12.debian.tar.xz"
+    url "https://deb.debian.org/debian/pool/main/libq/libquicktime/libquicktime_1.2.4-12.debian.tar.xz"
+    mirror "https://deb.debian.org/debian/pool/main/libq/libquicktime/libquicktime_1.2.4-12.debian.tar.xz"
```
Comment (fxcoudert) [inline Formula/libquicktime.rb]: Remove `mirror`

File: Formula/logcheck.rb [hunk 14]
```diff
@@ -1,8 +1,8 @@
 class Logcheck < Formula
   desc "Mail anomalies in the system logfiles to the administrator"
   homepage "https://packages.debian.org/sid/logcheck"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/l/logcheck/logcheck_1.3.19.tar.xz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/l/logcheck/logcheck_1.3.19.tar.xz"
+  url "https://deb.debian.org/debian/pool/main/l/logcheck/logcheck_1.3.19.tar.xz"
+  mirror "https://deb.debian.org/debian/pool/main/l/logcheck/logcheck_1.3.19.tar.xz"
```
Comment (fxcoudert) [inline Formula/logcheck.rb]: Remove `mirror`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by nsuchy** [comment]
Hi @fxcoudert thank you for reviewing this. I've made the changes you requested. Let me know if any further changes are necessary.

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**